quinn-proto/src/crypto.rs

```diff
@@ -79,6 +78,24 @@ impl TlsSession {
             },
         }
     }
+
+    pub fn write_handshake(&mut self, buf: &mut Vec<u8>) -> Result<Option<Crypto>, TLSError> {
+        match self.write_hs(buf) {
+            Some(secrets) => match self.get_negotiated_ciphersuite() {
+                None => Ok(None),
```

Comment:
We shouldn't ever be throwing away secrets rustls provides us; this should probably be unwrapped instead.

Reply:
I changed it to `panic!()`, which seems warranted for the case that rustls gives us secrets but not a cipher suite.